virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -194,73 +201,48 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
+
+//4.15 still unresolved 
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
 ** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
-// Leave in for 4.14 per Stu; remove from 4.15 RNs and move content to CNV docs, per Avital
-* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2157951[*BZ#2157951*])
+//4.15 Keep this per Stu, ETA for fixing this issue is 4.18. 
+* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. 
 +
 Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to FIPS servers running on RHEL 9. Similarly, RHEL 9 clients in FIPS mode cannot connect to servers that only support TLS 1.2 without EMS. This in practice means that these clients cannot connect to servers on RHEL 6, RHEL 7 and non-RHEL legacy operating systems. This is because the legacy 1.0.x versions of OpenSSL do not support EMS or TLS 1.3. For more information, see link:https://access.redhat.com/solutions/7018256[TLS Extension "Extended Master Secret" enforced with Red Hat Enterprise Linux 9.2].
 
-** As a workaround, upgrade legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
-
-[discrete]
-[id="virt-4-15-ki-webconsole"]
-==== Web console
-//CNV-34771 from Ying
-* If you upgrade {product-title} 4.13 to 4.14 without upgrading {VirtProductName}, the Virtualization pages of the web console crash. (link:https://bugzilla.redhat.com/show_bug.cgi?id=OCPBUGS-22853[*OCPBUGS-22853*])
-
-** As a workaround, you must upgrade the {VirtProductName} Operator to 4.14 manually or set your subscription approval strategy to *Automatic*.
\ No newline at end of file
+** As a workaround, upgrade legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
```

Comment:
Update is the preferred term per our [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
** As a workaround, update legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
```